02-library/cs506/sim.py

```
@@ -1,13 +1,52 @@
+import math
+
 def euclidean_dist(x, y):
-    raise NotImplementedError()
+    if len(x) != len(y):
+        return "Can't take similarity of vectors in different dimensions"
```

It's better to raise an error than return an inconsistent data type